### PR TITLE
Collect SystemInfo for Pods and ECS Tasks

### PR DESCRIPTION
**Pull request by aureleoules**
### What does this PR do?

Tested on staging on pods and ecs tasks.

### CODE REVIEW

**Review by cswatt — APPROVED** [review]
relnote lgtm

File: go.mod [hunk 1]
```diff
@@ -596,7 +596,7 @@ require github.com/lorenzosaino/go-sysctl v0.3.1
 
 require (
 	github.com/DATA-DOG/go-sqlmock v1.5.2
-	github.com/DataDog/agent-payload/v5 v5.0.127
+	github.com/DataDog/agent-payload/v5 v5.0.129-0.20240731155525-0096eb7031ac
```
Comment (fisherevans) [inline go.mod:599]: just commenting as a reminder to bump to an official release before merging

**Review by fisherevans — APPROVED** [review]
simple enough

**Pull request merged**